src/test/java/org/apache/commons/beanutils2/converters/ConverterTestSuite.java

```diff
@@ -53,7 +53,6 @@ public static Test suite() {
         testSuite.addTestSuite(SqlDateConverterTestCase.class);
         testSuite.addTestSuite(SqlTimeConverterTestCase.class);
         testSuite.addTestSuite(SqlTimestampConverterTestCase.class);
-        testSuite.addTestSuite(URLConverterTestCase.class);
```

Comment:
FYI: This whole class is no longer needed since we've been on Maven for a while... I'll remove it later.